client/state/login/magic-login/actions.js

```diff
@@ -111,3 +111,7 @@ export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
 			} );
 		} );
 };
+
+export function goBackToWordPressDotCom() {
```

Comment:
If we decide to omit `actionCallback` above, this will not be needed. 

I'm not sure if `state` is the best place to put this in, since this doesn't directly work with the Redux state.